src/components/ReportActionItem/MoneyRequestPreview.js

```diff
@@ -233,7 +233,12 @@ function MoneyRequestPreview(props) {
                 errorRowStyles={[styles.mbn1]}
                 needsOffscreenAlphaCompositing
             >
-                <View style={[isScanning || props.isWhisper ? styles.reportPreviewBoxHoverBorder : undefined]}>
+                <View
+                    style={[
+                        isScanning || props.isWhisper ? {...styles.reportPreviewBoxHoverBorder, ...styles.reportContainerBorderRadius} : undefined,
+                        !props.onPreviewPressed ? [styles.moneyRequestPreviewBox, ...props.containerStyles] : {},
```

Comment:
@fedirjh `onPreviewPressed` is an optional prop, before if onPreviewPressed is not passed the styling will not be applied, it thought it will prevent a regression. when there is no function is passed for that prop.

https://github.com/Expensify/App/blob/cb4e665533f6dce86ef1d14aa1cadaef5b7c74df/src/components/ReportActionItem/MoneyRequestPreview.js#L333-L335

Reply:
You can remove this line to test this change

https://github.com/Expensify/App/blob/a4603d6d3d492d422082cac073fc6cfa5116a127/src/components/ReportActionItem/MoneyRequestAction.js#L142


https://github.com/Expensify/App/blob/a4603d6d3d492d422082cac073fc6cfa5116a127/src/components/ReportActionItem/MoneyRequestPreview.js#L49C1-L50